webpack.config.js

```diff
@@ -0,0 +1,25 @@
+const path = require('path');
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+
+module.exports = {
+  entry: './src/js/index.js',
+  mode: 'development',
+  plugins: [
```

Comment:
- [ ] OPTIONAL
Kindly specify a `dev Sever` in your `Webpack.config.js` like this:

**Add a devServer to your Webpack:**
![image](https://user-images.githubusercontent.com/12745474/134181137-5d6d1c26-3623-4f21-9884-ce5150901041.png)

Reply:
Thanks. The dev server has been included.